app/views/admin/rebate_forms/_form.html.haml

```diff
@@ -1,8 +1,15 @@
 = form_for [:admin, @rebate_form] do |f|
-  = f.label :valuation_id
-  = f.text_field :valuation_id
-  = f.label :token
-  = f.text_field :token
+  - show_fields = ['income','dependants', 'full_name', 'lived_here_before_july_2018', 'has_home_business', 'email', 'phone_number', 'email_phone_can_be_used']
+  - @rebate_form.fields.each do |item|
+    - show_fields.each do |hf|
+      - if hf == item.first
+        = f.label item.first
+        = f.text_field(item.first, :value => item.last)
+
+  = f.hidden_field(:updated_by, :value => current_user.email)
```

Comment:
I don't think this should be on the form. It's not something we ask the browser to send us.
Instead, on the controller read who is the current user and use that

Otherwise i'll send from my browser that the email is someone else.